robot_skills/src/robot_skills/arm/grasp_position_detector.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+import rospy
+from sensor_msgs.msg import JointState
+
+from robot_skills.robot_part import RobotPart
+
+
+class GraspPositionDetector(RobotPart):
+    def __init__(self, robot_name, tf_buffer, wrench_topic):
+        """
+        Class for detecting whether or not the robot is holding something
+
+        :param robot_name: Name of the robot
+        :param tf_buffer: tf2_ros.Buffer for use in RobotPart
+        :param wrench_topic: Topic to use for measurement
```

Comment:
wrench was used in the previous grasp detector. This one is more of a joint topic